.run/Run environmentd.run.xml

```diff
@@ -0,0 +1,22 @@
+<component name="ProjectRunConfigurationManager">
+  <configuration default="false" name="Run environmentd" type="CargoCommandRunConfiguration" factoryName="Cargo Command">
+    <option name="buildProfileId" value="dev" />
+    <option name="command" value="run --bin environmentd -- --startup-log-filter=warn --system-parameter-default=log_filter=warn --internal-http-listen-addr=0.0.0.0:6878 --orchestrator=process --orchestrator-process-secrets-directory=$PROJECT_DIR$/mzdata/secrets --orchestrator-process-tcp-proxy-listen-addr=0.0.0.0 --orchestrator-process-prometheus-service-discovery-directory=$PROJECT_DIR$/mzdata/prometheus --orchestrator-process-scratch-directory=$PROJECT_DIR$/scratch --secrets-controller=local-file --persist-consensus-url=postgres://root@localhost:26257/materialize?options=--search_path=consensus --persist-blob-url=file://$PROJECT_DIR$/mzdata/persist/blob --timestamp-oracle-url=postgres://root@localhost:26257/materialize?options=--search_path=tsoracle --environment-id=local-az1-a1e5718e-9753-483c-9cb0-fe8f9e715f24-0 --bootstrap-role=materialize   " />
```

Comment:
Is this identical to the params passes when running `bin/environmentd`?

Reply:
Ah thanks for making me check. I had left in some additional filters I use locally.

Update from a fresh cli run:
```
environmentd --internal-http-listen-addr=0.0.0.0:6878 --orchestrator=process --orchestrator-process-secrets-directory=/Users/siddharthasahu/Downloads/installations/materialize/mzdata/secrets --orchestrator-process-tcp-proxy-listen-addr=0.0.0.0 --orchestrator-process-prometheus-service-discovery-directory=/Users/siddharthasahu/Downloads/installations/materialize/mzdata/prometheus --orchestrator-process-scratch-directory=/Users/siddharthasahu/Downloads/installations/materialize/scratch --secrets-controller=local-file --persist-consensus-url=postgres://root@localhost:26257/materialize?options=--search_path=consensus --persist-blob-url=file:///Users/siddharthasahu/Downloads/installations/materialize/mzdata/persist/blob --timestamp-oracle-url=postgres://root@localhost:26257/materialize?options=--search_path=tsoracle --environment-id=local-az1-77fedbe0-a02c-479f-8252-398c66aae4f5-0 --bootstrap-role=materialize
```